### PR TITLE
contract deal making support

### PR DESCRIPTION
**Pull request by shrenujbansal**
## Checklist

Before you mark the PR ready for review, please make sure that:

- [ ] Commits have a clear commit message.
- [ ] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: cmd/lotus-shed/cid.go [hunk 1]
```diff
@@ -94,6 +95,25 @@ var cidIdCmd = &cli.Command{
 	},
 }
 
+var cborSerCid = &cli.Command{
+	Name:      "cborSer",
```
Comment (magik6k) [inline cmd/lotus-shed/cid.go:99]: `cbor` (or `to-cbor`)?

File: build/params_2k.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ const GenesisFile = ""
 
 var NetworkBundle = "devnet"
 var BundleOverrides map[actorstypes.Version]string
-var ActorDebugging = true
```
Comment (magik6k) [inline build/params_2k.go:24]: Why change?

File: storage/pipeline/input.go [hunk 3]
```diff
@@ -349,13 +350,20 @@ func (m *Sealing) SectorAddPieceToAny(ctx context.Context, size abi.UnpaddedPiec
 		m.inputLk.Lock()
 	}
 
+	fmt.Println("!!!!!!!!!!!!!!! BEFORE ADD PENDING PIECE!!!!!!!!!!!!!!!!!!!!!")
```
Comment (magik6k) [inline storage/pipeline/input.go:353]: Debug stuff

**Comment by ZenGround0** [comment]
@shrenujbansal in light of the direction you went with integration into boost can we close this?

**Comment by shrenujbansal** [comment]
Deal making support moved into boost

**Pull request closed without merging**